### ISSUE TITLE
End of Java 11 support

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#2305]
## Overview

Jpsonic's Java11Support will end when v112.2.0 is released.

### Detail

Now is a compelling time to end support for Java 11.

 - The end of Active Support for Java11 in Sep 2023
 - Required to prepare for [Spring Boot 3.0](https://github.com/tesshucom/jpsonic/issues/839)
   - Open source support for Spring 2.x will end in Nov 2023
 - Required for [DWR upgrade](https://github.com/tesshucom/jpsonic/issues/1630)
 - [JDK21 GA release in September 2023](https://openjdk.org/projects/jdk/21/).
   - For Oracle, this version is the next LTS. In other words, Java 11 has come to an end, and it is time to search for the next mainstream.
 - Jpsonic's Docker is steadily improving. It will continue to improve. 
   - In other words, it can be used even on OSs that do not provide the latest Java.

![image](https://github.com/tesshucom/jpsonic/assets/27724847/c21cb1a2-0e30-4dda-b6d0-b1ba07468a85)
